torchvision/datapoints/_bounding_box.py

```diff
@@ -98,6 +98,11 @@ def wrap_like(
     def __repr__(self, *, tensor_contents: Any = None) -> str:  # type: ignore[override]
         return self._make_repr(format=self.format, spatial_size=self.spatial_size)
 
+    def __deepcopy__(self, memo: Dict[int, Any]) -> BoundingBox:
+        # We don't need to deepcopy the metadata, i.e. `format` and `spatial_size` here, since enums and tuples are
+        # immutable and thus `copy.deepcopy` is a no-op
+        return super().__deepcopy__(memo)
```

Comment:
I'm fine with adding one extra function call here that basically does nothing, but allows us to place the comment in the right place. LMK if you think otherwise.

Reply:
Sounds OK.

We *could* add the call if we wanted to be 100% safe and prevent a future issue if we ever were to change the types to something mutable. But that would probably be BC-breaking anyway.

Reply:
wait so why do we need to override `__deepcopy__` at all here?

Reply:
As discussed offline, I've removed the method here and added the comment to the superclass method.